Cubical/Data/Fin/Base.agda

```diff
@@ -69,3 +72,24 @@ elim P fz fs {suc k} fj
   ; (inr (fk , p)) → subst P p (fs (elim P fz fs fk))
   }
 
+any?
+  : ∀ {n} {P : Fin n → Type ℓ}
+  → (∀ i → Dec (P i))
+  → Dec (Σ (Fin n) P)
+any? {n = zero}  {P = _} P? = no (λ (x , _) → ¬Fin0 x)
+any? {n = suc n} {P = P} P? =
+  mapDec
+    (λ
+      { (inl P0) → fzero , P0
+      ; (inr (x , Px)) → fsuc x , Px
+      }
+    )
+    (λ n h → n (helper h))
+    (P? fzero ⊎? any? (P? ∘ fsuc))
+  where
+    helper
+      : Σ (Fin (suc n)) P
+      → (P fzero ⊎ Σ (Fin n) (λ z → P (fsuc z)))
```

Comment:
You don't need parentheses around the conclusion. Also, why not put the type on one line? Same applies to the toplevel type.